code/modules/power/singularity/particle_accelerator/particle.dm

```diff
@@ -27,32 +29,39 @@
 	source = null
 	movetotarget = 1
 
+
 /obj/effect/accelerated_particle/weak
 	movement_range = 8
-	energy = 5
+	energy = PARTICLE_ENERGY / 2
 	icon_state="particle0"
-	resetVariables()
-		..("energy", "movement_range")
-		movement_range = 8
-		energy = 5
+
+/obj/effect/accelerated_particle/weak/resetVariables()
```

Comment:
Why are these override here? Why would you need to explicitly reset these variables?